teleport-node/update-necocli.sh

```diff
@@ -15,10 +15,14 @@ FILE="neco-operation-cli-linux_${DATE}_amd64.deb"
 if [ ! -f ${HOME}/deb/${FILE} ]; then
     echo "Downloading and extracting Neco CLI tools..."
     curl -sLf -O https://github.com/cybozu-go/neco/releases/download/release-${DATE}/${FILE}
-    mkdir -p ${HOME}/neco-operation-cli
-    dpkg -x ${FILE} ${HOME}/neco-operation-cli
-    mkdir -p ${HOME}/deb
-    mv ${FILE} ${HOME}/deb/
+    if [ $? -eq 0 ]; then
```

Comment:
Why don't we retry `curl` several times?

Reply:
@ymmt2005 Because we agreed not to retry `curl` command here. Do you think it is beneficial?

Reply:
Ah the intention is obvious from the text. I'll fix it.